bigcode_eval/tasks/humaneval.py

```diff
@@ -48,14 +48,12 @@ class GeneralHumanEval(Task):
     DATASET_PATH = "openai_humaneval"
 
     def __init__(
-            self, 
-            strip_prompt, 
-            k=[1, 10, 100], 
-            num_workers=16, 
-            timeout=3.0, 
-            humaneval_one_shot=False, 
-            humaneval_prompt_quality=2, 
-            humaneval_add_context=False
+            self,
+            strip_prompt,
+            nuggets_config,
```

Comment:
Let's add this new argument to the end, we can give it a default parameter value = `None`